source/billing_system.html.md.erb

```diff
@@ -0,0 +1,74 @@
+# The GOV.UK PaaS billing system
+
+The GOV.UK Platform as a Service (PaaS) billing system generates bills for GOV.UK PaaS tenants. Our tenants use GOV.UK PaaS to create and delete various services and resources, such as compute instances running applications and databases. We need to calculate how much these services and resources cost on a monthly basis so we can recoup this money from tenants.
+
+The GOV.UK PaaS billing system comprises:
+
+- 2 applications written in Golang: the `paas-billing-collector` and the `paas-billing-api`
```

Comment:
Might be worth saying that both applications use the same codebase?

Reply:
Good shout! Added in a comment and link to the paas-billing repo